R/pretty_lollipop_plot.R

```diff
@@ -184,34 +332,114 @@ pretty_lollipop_plot <- function(
             data = domain_data, 
```

Comment:
I think that the protein model should be a separate plot from the lollipops, and then they should be arranged together. When you have a maf file that has many mutations (it's not uncommon to get > 20-30 mutations in a sample) the protein model gets really compressed. See for example this [Slack message](https://morinlabsfu.slack.com/archives/C06GR6YN7AA/p1734108232187889?thread_ts=1734107400.345649&cid=C06GR6YN7AA).